src/libcore/slice.rs

```diff
@@ -655,6 +655,12 @@ impl<'a, T> Default for &'a [T] {
     fn default() -> &'a [T] { &[] }
 }
 
+#[stable(feature = "rust1", since = "1.5.0")]
+impl<'a, T> Default for &'a mut [T] {
+    #[stable(feature = "rust1", since = "1.5.0")]
```

Comment:
I don't believe the method itself needs/desires the annotation (you can't partially-stabley implement a trait). The fact that the `&[T]` impl does is basically cargo-culting because these annotations are useless today.

@petrochenkov can probably best speak to what the heck the "right" thing is here, though.